### DISCUSSION TITLE
What is Actions button in the middle of the admin page?

### DISCUSSION THREAD

**Discussion by 94929** [aminalaee/sqladmin#331] [Q&A] [answered]
![CleanShot 2022-09-24 at 20 13 05](https://user-images.githubusercontent.com/10961008/192094778-156576b3-76bd-4ca3-a10e-a0f9e704122c.png)
As seen on the screenshot, I have an `Actions` button in the middle of the admin page.
However, the official demo https://sqladmin-demo.aminalaee.dev/admin/profile/list does not include that.
Does anyone know how to get rid of this?
Thanks in advance.

**Comment by aminalaee** [comment, marked as answer]
I need to update the demo app.
Right now it’s only to bulk delete items hut in future you can define custom actions there too.

**Reply by 94929** [reply]
Hi, thanks for the reply. Is there an option to remove the button?

**Reply by aminalaee** [reply]
Right now there's no way other than overriding the templates.

But I see another issue here that you have set `can_delete=False` but the bulk delete is still there. That I can fix now.

**Reply by 94929** [reply]
Yep, I have set the `can_delete=False` but the delete action is there and causing error when clicked. Would appreciate if bulk delete action is remove (not the actions button itself).
